.github/workflows/deploy.yml

```
@@ -21,10 +21,11 @@ jobs:
           python-version: '3.10'
 
       - name: Install mamba
-        uses: mamba-org/provision-with-micromamba@main
+        uses: mamba-org/setup-micromamba@v1
         with:
-          micromamba-version: '1.4.2'
-          environment-file: 'build-environment.yml'
+          micromamba-version: '1.5.0'
```

The CI fails with a cryptic error.

Wondering if this could be related to the format of the version. The `mamba-org/setup-micromamba` README mentions a `-0` suffix:

```suggestion
          micromamba-version: '1.5.0-0'
```